microcosm_pubsub/context.py

```diff
@@ -27,43 +26,39 @@ def __init__(self, graph):
         self.enable_ttl = graph.config.sqs_message_context.enable_ttl
         self.initial_ttl = graph.config.sqs_message_context.initial_ttl
 
-    def __call__(self, message, **kwargs):
+    def __call__(self, context, **kwargs) -> Dict[str, str]:
```

Comment:
Nit: annotate function args (this can explicitly label it as a `Union` for now, if you're not immediately fixing the `dict` case).